database/testhelpers/test_helpers.go

```diff
@@ -52,15 +52,15 @@ func GetConnectionDetails() *database.Config {
 	}
 
 	return &database.Config{
-		Enabled:           true,
-		Driver:            "postgres",
+		Enabled: true,
+		Driver:  "postgres",
 		ConnectionDetails: drivers.ConnectionDetails{
-			// Host:     "",
-			// Port:     5432,
-			// Username: "",
-			// Password: "",
-			// Database: "",
-			// SSLMode:  "",
+			Host:     "localhost",
+			Port:     5432,
+			Username: "andrew",
```

Comment:
🥇

Reply:
:D i think this is only the 6th time i have forgot to revert these maybe one day i will remember